addons/default/visiosoft/defaultadmin-theme/resources/views/login.twig

```diff
@@ -35,14 +35,14 @@
 
                 <div class="form-group">
                     {% if setting_value('anomaly.module.users::login') == 'username' %}
-                        {{ form.fields.username.setPlaceholder(form.fields.username.label).addAttribute('autofocus', 1).input|raw }}
+                        {{ form.fields.username.setValue(app.request.get('username')).setPlaceholder(form.fields.username.label).addAttribute('autofocus', 1).input|raw }}
```

Comment:
if there's no username send it's working right? @diashalabi

Reply:
yes